docs/windows.md

```diff
@@ -313,12 +313,17 @@ Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
 Firstly, install wins, kubelet, kubeadm using script `PrepareNode.ps1` provided
 by kubernetes. The third component [`wins`](https://github.com/rancher/wins) is
 used to run kube-proxy and antrea-agent on Windows host inside the Windows
-container.
+container. Specify the Node IP, Kubernetes Version and Container Runtime while
+running the script. If you do not specify any container runtime, it will be Docker
+by default. (Note: Docker has been deprecated since k8s version 1.24). You can also
+specify whether you want to install OVS on the node.
 
```

Comment:
Add introduction for kube-proxy installation.

Reply:
We should add clear command examples to explain how to run Prepare-Node.ps1 for docker/containerd testbed with/without kube-proxy. @NamanAg30

Reply:
change made

Reply:
Reminder